src/api/routes.py

```diff
@@ -32,21 +48,29 @@ async def index_name(request: Request):
 @router.post("/chat/stream")
 async def chat_stream_handler(
     chat_request: ChatRequest,
+    chat_client : ChatCompletionsClient = Depends(get_chat_client),
+    model_deployment_name : str = Depends(get_chat_model),
+    search_index_manager: SearchIndexManager = Depends(get_search_index_namager)
 ) -> fastapi.responses.StreamingResponse:
-    chat_client = globals["chat"]
     if chat_client is None:
         raise Exception("Chat client not initialized")
 
     async def response_stream():
         messages = [{"role": message.role, "content": message.content} for message in chat_request.messages]
-        model_deployment_name = globals["chat_model"]
         
-        prompt_messages = globals["prompt"].create_messages()
+        prompt_messages = PromptTemplate.from_string('You are a helpful assistant').create_messages()
         # Use RAG model, only if we were provided index and we have found a context there.
-        if globals["rag"] is not None:
-            context = await globals["rag"].search(chat_request)
+        if search_index_manager is not None:
+            context = await search_index_manager.search(chat_request)
             if context:
-                prompt_messages = globals["rag_prompt"].create_messages(data=dict(context=context))
+                prompt_messages = PromptTemplate.from_string(
+                    'You are a helpful assistant that answers some questions '
+                    'with the help of some context data.\n\nHere is '
+                    'the context data:\n\n{{context}}').create_messages(data=dict(context=context))
+                # Remove this line.
```

Comment:
Is the comment correct "Remove this line."?

Reply:
Fixed.